src/main/resources/application.yaml

```
@@ -1,76 +1,80 @@
-    # link: https://www.data.go.kr/data/15084084/openapi.do
-    vilage-fcst-info-service:
-      first-path: /1360000/VilageFcstInfoService_2.0
-      # 초단기실황조회
-      get-ultra-srt-ncst:
-        second-path: /getUltraSrtNcst
-        date-type: JSON
-      # 예보버전조회
-      get-fcst-version:
-        second-path: /getFcstVersion
-        date-type: JSON
+spring:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)